static/js/vue-cdr-access/src/components/browseSort.vue

```diff
@@ -0,0 +1,76 @@
+<template>
+    <div class="browse-sort select is-medium">
+        <select @change="sortRecords" v-model="sort_order">
+            <option value="">Sort by...</option>
+            <option value="title-asc">Title A-Z</option>
+            <option value="title-desc">Title Z-A</option>
+            <option value="added-desc">Date Created (newest)</option>
+            <option value="added-asc">Date Created (oldest)</option>
+        </select>
+    </div>
+</template>
+
+<script>
+    export default {
+        name: 'browseSort',
+
+        props: {
+            records: Array
+        },
+
+        data() {
+            return {
+                sort_order: ''
+            }
+        },
+
+        methods: {
+            sortRecords() {
+                let sort_values = this.sort_order.split('-');
```

Comment:
If i'm understanding correctly, the sort in this component is doing an in-memory sort rather than asking the server for sorted paginated results? While that is probably fine for admin sets since they don't have many children, other types like folders and collections will have too many children to pass to the client. Could this component be changed to request data from the server instead?

Reply:
Yes, it could